README.md

```diff
@@ -1,135 +1,338 @@
-## <img src="doc/gambit.png" alt="gambit logo" height="40" align="left"> Gambit: Certora's Mutation Generator for Solidity
+# Generating Mutations
 
 This is a mutation generator for Solidity.
-Mutation Testing is a technique for
-  evaluating and improving test suites or specifications used
-  for testing or verifiying Solidity smart contracts.
+It takes as input a solidity source file (or a configuration file as you can see below)
+and produces a set of uniquely mutated solidity source files which are output in the `out/` directory by default.
+The source is [publicly available](https://github.com/Certora/gambit).
```

Comment:
I think we can add a link here to the doc: https://docs.certora.com/en/latest/docs/gambit/index.html and indicate that more information on how to use it with the Certora Prover can be found there.